src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala

```diff
@@ -63,4 +63,28 @@ object WsClient extends LazyLogging {
     response
   }
 
+  def post(uri: String, headers: Map[String, String], json: JsValue): StandaloneWSResponse = {
+    println("")
+    logger.info(s"POST request URI: $uri")
+    logger.info(s"POST request headers: $headers")
```

Comment:
Can you keep the URI as **info** and keep the headers and json as **debug**. 
We don't want to print everything for every request